src/hotspot/share/prims/jvmtiEnvBase.hpp

```diff
@@ -371,7 +391,10 @@ class SetFramePopClosure : public JvmtiHandshakeClosure {
       _env(env),
       _state(state),
       _depth(depth) {}
-  void do_thread(Thread *target);
+  void do_thread(Thread *target) {
+    doit(target, false);
```

Comment:
Please add a comment after the true parameter to
indicate the name of the doit() function's parameter,
e.g., `false /* self */`.